internal/upvs/config.go

```diff
@@ -6,12 +6,14 @@ type Configuration struct {
 	Url                string
 	ApiTokenPrivateKey string
 	OboTokenPublicKey  string
+	SsoSubject         string
```

Comment:
add comment above based on what we should set it, and whhy we need it/where we use it